docs/judge/problem_format.md

```diff
@@ -16,11 +16,11 @@ The batch will contain the keys `points` and `batched`. `batched` will map to a
 An `init.yml` object can contain a top-level `grader` node, which contains a path to a Python file to be executed as a grader for the problem. The grader has access to the archive specified in `archive`. A `grader` Python script must implement a function that is called by the judge:
 
 ```python
-def grade(case, process, case_input=None, case_output=None, point_value=0, **kwargs)
+def grade(self, case):
```

Comment:
This is actually implemented by defining a `Grader` that takes `__init__(judge, problem, language, source)`. This `Grader` typically inherits `StandardGrader` (https://github.com/DMOJ/judge/blob/master/dmoj/graders/standard.py)